packages/bot-web-ui/src/components/dashboard/bot-builder/toolbox/toolbox.tsx

```diff
@@ -56,8 +58,14 @@ const Toolbox = ({
         <div className='dashboard__toolbox'>
             <div id='gtm-toolbox' className='db-toolbox__content'>
                 <div className='db-toolbox__header'>
-                    <div className='db-toolbox__title' onClick={() => setOpen(!is_open)}>
-                        {localize('Blocks menu')}
+                    <div
+                        className='db-toolbox__title'
+                        onClick={() => {
+                            setOpen(!is_open);
+                            setVisibility(false);
+                        }}
+                    >
+                        {localize('Blocks menu d')}
```

Comment:
I think there is typo, please fix it @rupato-deriv 

it should be `Blocks menu`

Reply:
done ✅  thanks